core/src/main/scala/cromwell/core/ExecutionStatus.scala

```diff
@@ -5,6 +5,20 @@ object ExecutionStatus extends Enumeration {
   val NotStarted, QueuedInCromwell, Starting, Running, Failed, RetryableFailure, Done, Bypassed, Aborted = Value
   val TerminalStatuses = Set(Failed, Done, Aborted, RetryableFailure, Bypassed)
 
+  implicit val ExecutionStatusOrdering = Ordering.by { status: ExecutionStatus =>
+    status match {
+      case NotStarted => 0
+      case QueuedInCromwell => 1
+      case Starting => 2
+      case Running => 3
+      case Failed => 4
+      case RetryableFailure => 4
+      case Done => 4
+      case Aborted => 4
+      case Bypassed => 4
```

Comment:
Unlike `WorkflowState`, there are a lot of ties here that can lead to unpredictable results.